src/pudl/metadata/resources/ferc1.py

```diff
@@ -1896,24 +1926,28 @@
         "field_namespace": "ferc1",
     },
     "out_ferc1__yearly_detailed_balance_sheet_assets": {
-        "description": (
-            "This table contains granular accounting data from utilities' balance sheet assets. "
-            "This table is derived from four FERC Form 1 accounting tables with nested calculations:\n\n"
-            " * `core_ferc1__yearly_balance_sheet_assets_sched110`\n"
-            " * `core_ferc1__yearly_utility_plant_summary_sched200`\n"
-            " * `core_ferc1__yearly_plant_in_service_sched204`\n"
-            " * `core_ferc1__yearly_depreciation_by_function_sched219`\n\n"
-            "We reconciled the nested calculations within these tables and then identified the "
-            "most granular data across the tables.\n"
-            "We applied slight modifications to three columns (utility_type, plant_function & plant_status) "
-            "as compared to the originally reported values in our core tables. "
-            "The modifications were applied to either provide more specificity (i.e. we converted"
-            "some `total` utility_type's into `electric`) or to condense similar categories "
-            "for easier analysis (i.e. creating a `hydraulic_production` plant_function by "
-            "combining `hydraulic_production_conventional` and `hydraulic_production_pumped_storage`\n"
-            "See ``pudl.output.ferc1.Exploder`` for more details. This table was made entirely with "
-            "support and direction from RMI."
-        ),
+        "description": {
+            "additional_summary_text": "granular, de-duplicated accounting data of utilities' balance sheet assets.",
+            "additional_source_text": "(Schedule 110, 200, 204 and 219)",
```

Comment:
Sneaky missing plural
```suggestion
            "additional_source_text": "(Schedules 110, 200, 204 and 219)",
```